### PR TITLE
add an example for cpu-only case

### PR DESCRIPTION
**Pull request by runzhliu**
This example shows how to run a cpu-only mpijob.

### CODE REVIEW

File: examples/hovorod-cpu/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Horovod CPU-Only Case
+
+This example shows how to run a cpu-only mpijob.
+
+## How to Build Image
+
+Please download Horovod cpu only dockerfile and then build the image as follows:
+
+```bash
+curl -O https://raw.githubusercontent.com/horovod/horovod/master/Dockerfile.cpu
```
Comment (terrytangyuan) [inline examples/hovorod-cpu/README.md:10]: Can we copy the Dockerfile to this repo?

File: examples/hovorod-cpu/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# change the default tensorflow version from 2.1.0 to 1.14.0,
```
Comment (terrytangyuan) [inline examples/hovorod-cpu/README.md:11]: Any reason for this?

File: examples/hovorod-cpu/tensorflow-mnist.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            - pml
+            - ob1
+            - -mca
+            - btl
+            - ^openib
+            - python
+            - /examples/tensorflow_mnist.py
+            resources:
+              limits:
+                cpu: "1"
```
Comment (terrytangyuan) [inline examples/hovorod-cpu/tensorflow-mnist.yaml:40]: Remove unnecessary quotes

File: examples/hovorod-cpu/tensorflow-mnist.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      template:
+        spec:
+          containers:
+          - command:
+            - ""
+            image: horovod-cpu:latest
+            name: mpi-worker
+            resources:
+              limits:
+                cpu: "2"
```
Comment (terrytangyuan) [inline examples/hovorod-cpu/tensorflow-mnist.yaml:53]: Same here

File: examples/hovorod-cpu/README.md (hunk 1 above)
Comment (runzhliu) [reply examples/hovorod-cpu/README.md]: Yes, I have copied it.

File: examples/hovorod-cpu/README.md (hunk 2 above)
Comment (runzhliu) [reply examples/hovorod-cpu/README.md]: I think the [example](https://github.com/horovod/horovod/blob/master/examples/tensorflow_mnist.py) in horovod is based on TF 1.x, also I have tested about that, TF 2.0 isn't compatible.

File: examples/hovorod-cpu/tensorflow-mnist.yaml (hunk 3 above)
Comment (runzhliu) [reply examples/hovorod-cpu/tensorflow-mnist.yaml]: Done.

File: examples/hovorod-cpu/tensorflow-mnist.yaml (hunk 4 above)
Comment (runzhliu) [reply examples/hovorod-cpu/tensorflow-mnist.yaml]: Done.

File: examples/hovorod-cpu/Dockerfile.cpu [hunk 5]
```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+ENV TENSORFLOW_VERSION=1.14.0
+ENV PYTORCH_VERSION=1.4.0
+ENV TORCHVISION_VERSION=0.5.0
+ENV MXNET_VERSION=1.6.0
+
+# Python 2.7 or 3.6 is supported by Ubuntu Bionic out of the box
+ARG python=2.7
```
Comment (terrytangyuan) [inline examples/hovorod-cpu/Dockerfile.cpu:9]: Can we use Python 3.6 then? Have you tested with Python 3.6?

Comment (runzhliu) [reply examples/hovorod-cpu/Dockerfile.cpu]: Just run some tests with Python3.6, it works fine ;)

Comment (runzhliu) [reply examples/hovorod-cpu/Dockerfile.cpu]: I have set Python3.6 as the default  in Dockerfile.

**Pull request merged**